win32ss/user/ntuser/sysparams.c

```diff
@@ -253,7 +253,7 @@ SpiUpdatePerUserSystemParameters(VOID)
     gspv.ncm.iCaptionWidth = SpiLoadMetric(L"CaptionWidth", 19);
     gspv.ncm.iCaptionHeight = SpiLoadMetric(L"CaptionHeight", 19);
```

Comment:
Note also that the specified default values are different from those in the hivedef.inf . No idea which ones are correct (compare with Win2k3).

Reply:
![VirtualBox_Windows 2003_08_12_2019_12_00_48](https://user-images.githubusercontent.com/2107452/70383514-6bb33080-19b2-11ea-9a47-1bbae2020484.png)
I don't know well...

Reply:
strange... :thinking:

Reply:
![VirtualBox_Windows XP Japanese_08_12_2019_12_02_50](https://user-images.githubusercontent.com/2107452/70383535-c3519c00-19b2-11ea-9342-9980bb0b339b.png)
Those are negative values in Windows.

Reply:
They look to me like uninitialized invalid values.